src/linux/Packaging.Linux/build.sh

```diff
@@ -154,8 +154,9 @@ tar -czvf "$SYMTARBALL" * || exit 1
 popd
 
 # Build .deb
-INSTALL_TO="$DEBROOT/usr/bin/"
-mkdir -p "$DEBROOT/DEBIAN" "$INSTALL_TO" || exit 1
+INSTALL_TO="$DEBROOT/usr/local/bin/share/gcm-core/"
+LINK_TO="$DEBROOT/usr/bin/"
```

Comment:
```suggestion
LINK_TO="$DEBROOT/usr/local/bin/"
```

..if we're placing binaries in `/usr/local/share` then the symlinks should match and be in `/usr/local/bin`